src/Strategy/MockClientStrategy.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Http\Discovery\Strategy;
+
+use Http\Mock\Client as Mock;
+
+/**
+ * Find the Mock client.
+ *
+ * @internal
```

Comment:
Is **this strategy** really internal? The others are, but it is meant that this class should be used by others, right?